### DISCUSSION TITLE
Add an option for users to decide the priority of highlights

### DISCUSSION THREAD

**Discussion by TigerWarz** [Chatterino/chatterino2#3872] [Ideas] [answered]
I feel like an option of some sorts that would allow users to choose which highlights are prioritised over others would be beneficial, as rather than the development team having influence over it themselves, each individual could decided instead.

This is mainly in response to the recent nightly build in which the highlight priority order has been change, to something that I personally do not prefer: having user highlights prioritised over the top of message/phrase based highlights.

I feel easiest implementation would be to have a default set, for example to how it is in the current nightly build, but allow users, within the highlights tab in settings, to set an integer value, 1 for top priority, 2 for middle, 3 for bottom, and set 0 to be to return to default priority

Attached below is a rough idea of how it would look like:
![chatterino thing](https://user-images.githubusercontent.com/80633637/180450330-9d464d61-6a97-44ec-8b9c-7b96cd64516c.png)

**Comment by pajlada** [comment, marked as answer]
Duplicate of #3865 

The way you've consider this being solved is how I also think this should be solved.
We might need to move highlight options to their own modal or something at the same time given how many special fields are visible

**Comment by pajlada** [comment]
Highlight order is modified by https://github.com/Chatterino/chatterino2/pull/4303 to message > user, which should hopefully lessen the need for a custom highlight order
